monokai-theme.el

```diff
@@ -4610,6 +4610,17 @@ Also affects 'linum-mode' background."
                                         :background ,monokai-256-background
                                         :inverse-video t))))
 
+   ;; realgud
+   `(realgud-overlay-arrow1
+     ((,monokai-class (:foreground ,monokai-green-d))
+      (,monokai-256-class  (:foreground ,monokai-green-d))))
```

Comment:
We need to use `,monokai-256-green/yellow/orange/-d` for the `,monokai-256-class` colours.